dabuild.in

```diff
@@ -23,6 +23,13 @@ if [ "${PWD%*/aports*}" = "$PWD" ]; then
   exit 1
 fi
 
+## allow setting of arch by env variable
+[ ! "$DABUILD_ARCH" ] && DABUILD_ARCH=$(uname -m)
```

Comment:
```suggestion
[ -z "$DABUILD_ARCH" ] && DABUILD_ARCH=$(uname -m)
```

Reply:
What is the difference here?

Reply:
Functionally, these are the same I believe.

The `-z` is just more explicit about what you are testing for. `!` negates the implicit `-n` test on the string.